### ISSUE TITLE
Investigate GraalVM for JDK 21 breakage

### ISSUE THREAD

**Issue by Berstanio** [Berstanio/gdx-graalhelper#6]
```
Error: Feature defined by com.anyicomplex.gdx.lwjgl3.svm.ReflectionRegistrationFeature unexpectedly failed with a(n) java.lang.RuntimeException. Please report this problem to the authors of com.anyicomplex.gdx.lwjgl3.svm.ReflectionRegistrationFeature.
```

_Originally posted by @ThanosFisherman in https://github.com/Berstanio/gdx-graalhelper/issues/5#issuecomment-1800272302_

**Comment by Berstanio** [comment]
@ThanosFisherman
The issue should be resolved, can you give the 1.12.1.1-SNAPSHOT version a try?
Or the https://github.com/Berstanio/gdx-graalhelper/commit/4528686364319503aaaba8021418a0b00bd0c92b commit, if you are using jitpack.

**Comment by ThanosFisherman** [comment]
Hi @Berstanio 
Indeed it worked like a charm now with GraalVM 21! Great work :)
